### PR TITLE
Clean up of Microsoft.ApplicationModel.Resources.PriGen.targets (round 4)

### PR DESCRIPTION
**Pull request by rohanp-msft**
Remove more unused MSBuild code.

Microsoft.ApplicationModel.Resources.PriGen.targets was created from Microsoft.AppxPackage.targets where a bunch of appx generation MSBuild script code was removed (Microsoft.AppxPackage.targets does more than just PRI generation, which is all that we want Microsoft.ApplicationModel.Resources.PriGen.targets to do, as the name suggests). At that point though, not all non-PRI gen related MSBuild script code was removed. I'll now try to remove it over several rounds. This is round 4.

Round 1: #664
Round 2: #696
Round 3: #704 

How verified
Used the MRTCore C# sample app and lib. Got rid of build outputs. Replaced the prigen targets file in the nuget package install with the one in this change. Built and ran the app.

### CODE REVIEW

**Comment by huichen123** [comment]
>   </PropertyGroup>

is this needed?

---
Refers to: dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:111 in 9ba2918. [](commit_id = 9ba2918d4bf26ec6ffc218d0702b718f008c0903, deletion_comment = False)

**Comment by huichen123** [comment]
>   </PropertyGroup>

this?

---
Refers to: dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:121 in 9ba2918. [](commit_id = 9ba2918d4bf26ec6ffc218d0702b718f008c0903, deletion_comment = False)

**Comment by rohanp-msft** [comment]
>   </PropertyGroup>

nope. i can get rid of it in the next round, if that's ok.

---
In reply to: [828027482](https://github.com/microsoft/ProjectReunion/pull/770#issuecomment-828027482) [](http://example.com/codeflow?ancestors=828027482,828027482)

---
Refers to: dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:121 in 9ba2918. [](commit_id = 9ba2918d4bf26ec6ffc218d0702b718f008c0903, deletion_comment = False)

**Comment by rohanp-msft** [comment]
>   </PropertyGroup>

likely not. i need to look more carefully. i can get rid of it in the next round, if that's ok.

---
In reply to: [828027256](https://github.com/microsoft/ProjectReunion/pull/770#issuecomment-828027256) [](http://example.com/codeflow?ancestors=828027256,828027256)

---
Refers to: dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:111 in 9ba2918. [](commit_id = 9ba2918d4bf26ec6ffc218d0702b718f008c0903, deletion_comment = False)

**Comment by huichen123** [comment]
>   </PropertyGroup>

this is the 4th change of the cleanup. It becomes more difficult for next person trying to figure out what needs to be cleaned up. S/He needs to mentally merge all those changes together to get a full picture. Adding a few more just makes this worse.

---
In reply to: [828037384](https://github.com/microsoft/ProjectReunion/pull/770#issuecomment-828037384) [](http://example.com/codeflow?ancestors=828037384,828027256)

---
Refers to: dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:111 in 9ba2918. [](commit_id = 9ba2918d4bf26ec6ffc218d0702b718f008c0903, deletion_comment = False)

File: dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets [hunk 1]
```diff
@@ -1108,31 +992,6 @@
 
   </Target>
 
-  <!-- Expand content of PRI files. -->
-  <Target Name="_ExpandPriUploadFiles"
-          Condition="'$(BuildAppxUploadPackageForUap)' == 'true'">
-
-    <ItemGroup>
-      <_PriUploadFilesToExpand Include="@(_PriFilesFromPayload)"
```
Comment (huichen123) [inline dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:1116]: >_PriUploadFilesToExpand [](http://example.com/codeflow?start=7&length=23)

no need to create upload file?

Comment (rohanp-msft) [reply dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:1116]: yes. this isn't needed for us.

**Comment by rohanp-msft** [comment]
> > 
> 
> this is the 4th change of the cleanup. It becomes more difficult for next person trying to figure out what needs to be cleaned up. S/He needs to mentally merge all those changes together to get a full picture. Adding a few more just makes this worse.
> 
> In reply to: [828037384](https://github.com/microsoft/ProjectReunion/pull/770#issuecomment-828037384)
> 
> Refers to: dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:111 in [9ba2918](https://github.com/microsoft/ProjectReunion/commit/9ba2918d4bf26ec6ffc218d0702b718f008c0903). [](commit_id = [9ba2918](https://github.com/microsoft/ProjectReunion/commit/9ba2918d4bf26ec6ffc218d0702b718f008c0903), deletion_comment = False)

removed more things in second iteration including the two things Hui asked for in comments

File: dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets [hunk 2]
```diff
@@ -221,7 +204,6 @@
     <AppxDefaultResourceQualifiers Condition="'$(TargetPlatformIdentifierAdjusted)' == 'Windows' and '$(TargetPlatformVersion)' == '8.2'">$(AppxDefaultResourceQualifiers_Windows_82)</AppxDefaultResourceQualifiers>
```
Comment (huichen123) [inline dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:204]: we'd better check the version ahead of time, and never need this.

File: dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets (hunk 1 above)
Comment (huichen123) [reply dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:1116]: if you search "upload" in the file, are other hits needed?

File: dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets (hunk 2 above)
Comment (rohanp-msft) [reply dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:204]: I suspect AppxDefaultResourceQualifiers may not even be needed. I'll tackle that in the next round.

**Pull request merged**